### PR TITLE
Add a bit about Katrina

### PR DESCRIPTION
**Pull request by crichID**
This is for the WSC2018 workshop

✨ 

**bold**

@LizzHale 👋

### CODE REVIEW

**Comment by crichID** [comment]
comment

File: katrina-uychaco.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Lives in San Francisco, CA
+Here to teach Git & GitHub!
+Love all the amazing people!
```
Comment (crichID) [inline katrina-uychaco.txt:3]: Where is this?

File: katrina-uychaco.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Lives in San Francisco, CA
+Here to teach Git & GitHub!
```
Comment (LaMedusa) [inline katrina-uychaco.txt:2]: Where is "here"?

**Review by crichID — COMMENTED** [review]
this is amazing!

**Pull request merged**